perftestnotebook.py

```diff
@@ -210,6 +216,41 @@ def main():
         json.dump(ptnb.fmt_data, f, indent=4, sort_keys=True)
 
 
+    # Upload template file to Iodide
+    template = 'testing/template/template.txt'
+    tdata = ''
+    with open(template, 'r') as f:
+        tdata = f.read()
+
+    html = ''
+    with open('template_upload_file.html', 'r') as f:
+        html = f.read()
+
+    html = html.replace('replace_me', repr(tdata))
+    with open('upload_file.html', 'w+') as f:
+        f.write(html)
+    
+    webbrowser.open_new_tab('upload_file.html')
```

Comment:
Does this work when there is no browser open?

Reply:
yes it will automatically open a new browser and a new tab in it.